src/System.Text.Json/roadmap/images/diagram-1.txt

```diff
@@ -0,0 +1,22 @@
+' Licensed to the .NET Foundation under one or more agreements.
+' The .NET Foundation licenses this file to you under the MIT license.
+' See the LICENSE file in the project root for more information.
+' Requires PlantUML to interpret: http://plantuml.com/
+' See license info under http://plantuml.com/faq for more details
+
+@startuml
+package "Lowest JSON Layer" {
+  [JsonReader] #Yellow
```

Comment:
As discussed in the other PR, let's get rid of the UML.  (I'd drop the whole roadmap section, it's not something any other module has)

Reply:
I asked @ahsonkhan to add a markdown with a general description of the 5 components as a bunch of people commenting on the announcement weren't discovering this in corefxlab.